coffee-module/coffee-module-mp/coffee-module-mp-restclient/src/main/java/hu/icellmobilsoft/coffee/module/mp/restclient/provider/DefaultLoggerClientResponseFilter.java

```diff
@@ -138,8 +138,11 @@ protected String logEntity(ClientRequestContext requestContext, ClientResponseCo
         if (requestContext == null || responseContext == null) {
             return null;
         }
-        ByteArrayOutputStream out = new ByteArrayOutputStream();
         InputStream in = responseContext.getEntityStream();
+        if (in == null) {
+            return "";
```

Comment:
a tobbi helyen is null-al terunk vissza, akkor itt is az kellene

Reply:
Akkor a logolt üzenetben lenne egy `null` string. Inkább a fentebb lévő ellenőrzést írnám át pl exception dobásra.

Reply:
A jelen esetben a null ertek a helyes, mivel nincs entity. Mar az ures String is ertek - nem mondana igazat a funkcio

Reply:
Akkor viszont:
```java
return "entity: [null]";
```
?

Reply:
Kerdes hogy mi lenne a log hossz vezerlessel, inkabb a 
```java
return requestResponseLogger.printEntity(responseEntity, maxResponseEntityLogSize, RequestResponseLogger.RESPONSE_PREFIX);
```
formatumot kellene kovetni. De akkor az osszes eddigi return-t kellene modositani es mondjuk az exception elkapasnal a 
```java
        } catch (IOException e) {
            log.error("Error in logging response entity: " + e.getLocalizedMessage(), e);	            log.error("Error in logging response entity: " + e.getLocalizedMessage(), e);
            return null;	            return null;
        }	        }
    }
```
mar kerdeses mit kellene bele rakn...